DataFormats/RecoCandidate/src/classes_def.xml

```diff
@@ -82,9 +82,6 @@
   <class name="edm::helpers::KeyVal<edm::RefProd<std::vector<reco::Track> >,edm::RefProd<edm::OwnVector<reco::Candidate,edm::ClonePolicy<reco::Candidate> > > >" />
   <class name="edm::Wrapper<edm::AssociationMap<edm::OneToOne<std::vector<reco::Track>,edm::OwnVector<reco::Candidate,edm::ClonePolicy<reco::Candidate> >,unsigned int> > >" />
 
-  <class name="edm::Wrapper<reco::SimToRecoCollection>" />
-  <class name="edm::Wrapper<reco::RecoToSimCollection>" />
-
   <class name="reco::SimToRecoCollection" >
```

Comment:
Why was this working before the change in #26116 ?

I note belatedly, that the original definition of the typedef from a template is in DataFormats/RecoCandidate/interface/TrackAssociation.h 
So, while there is some generic logic to have a RECO+Sim to be in  SimDataFormats/TrackingAnalysis, the content of DataFormats/RecoCandidate/interface/TrackAssociation.h should be consistently in the same place.

It also looks pretty bad to have a Wrapper dictionary defined in SimDataFormats/TrackingAnalysis while the underlying object stays defined here in  DataFormats/RecoCandidate.

We should either revert the change in #26116 or complete the move.

Reply:
I think there was already a duplication in DataFormats/RecoCandidate/src/classes_def.xml  but as the duplication was with in same file/library that is why root did not complain. PR #26116 moved only the following in to  SimDataFormats/TrackingAnalysis
```
  <class name="edm::Wrapper<edm::AssociationMap<edm::OneToManyWithQualityGeneric<edm::View<reco::Track>,vector<TrackingParticle>,double,unsigned int,edm::RefToBaseProd<reco::Track>,edm::RefProd<vector<TrackingParticle> >,edm::RefToBase<reco::Track>,edm::Ref<vector<TrackingParticle>,TrackingParticle,edm::refhelper::FindUsingAdvance<vector<TrackingParticle>,TrackingParticle> > > > >" />
  <class name="edm::Wrapper<edm::AssociationMap<edm::OneToManyWithQualityGeneric<vector<TrackingParticle>,edm::View<reco::Track>,double,unsigned int,edm::RefProd<vector<TrackingParticle> >,edm::RefToBaseProd<reco::Track>,edm::Ref<vector<TrackingParticle>,TrackingParticle,edm::refhelper::FindUsingAdvance<vector<TrackingParticle>,TrackingParticle> >,edm::RefToBase<reco::Track> > > >" />
```

Reply:
I am fine with moving Wrapper back to DataFormats/RecoCandidate, we just need to teach duplicate dict checker to not complain about them.

Reply:
@smuzaffar would this be the only exception we have so far?

Reply:
yes. We can just remove `edm::Wrapper<edm::AssociationMap<....> ` (which were added by #26116)  from SimDataFormats/TrackingAnalysis and add back the above two definitions (duplicate dict checker will not complain these are just typedefs).

Reply:
please suggest if the above proposal sound OK?

Reply:
@smuzaffar for the time being yes please, let's restore RecoCandidate

Reply:
I would suggest to revert #26116 and start an IB. I will provide a new PR to fix this.

Reply:
> yes. We can just remove `edm::Wrapper<edm::AssociationMap<....> ` (which were added by #26116) from SimDataFormats/TrackingAnalysis and add back the above two definitions (duplicate dict checker will not complain these are just typedefs).

this would be fine for me.